expression/aggregation/aggregation.go

```diff
@@ -49,6 +50,9 @@ type Aggregation interface {
 	// GetResult will be called when all data have been processed.
 	GetResult(ctx *AggEvaluateContext) types.Datum
 
+	// SetResult2Chunk will be called when all data have been processed.
+	SetResultInChunk(chunk *chunk.Chunk, colIdx int, ctx *AggEvaluateContext)
```

Comment:
we need to two APIs to write result to Chunk:
1. for final result, we only write one column to a chunk
2. for partial result, we may write multiple results to a chunk

Reply:
ok